Gopkg.toml

```diff
@@ -101,3 +101,6 @@
 
 [[constraint]]
   name = "gopkg.in/yaml.v2"
+
+[[constraint]]
+  name = "github.com/stretchr/testify"
```

Comment:
This doesn't look like actively maintained because of pending PR and issues. Let make sure we shouldn't hit any blocker during the process when start using it.

Reply:
Yes, the last commit was on 5th august, thats 3 months of no activity. Since we already decided on using this on the issue i went ahead with this. Also with a quick google search i wasn't able to find any other actively maintained alternatives.
@minishift/minishift-dev Toughts on this?

Reply:
If we are going down this route, I would think we are going all in, right? I mean current tests are getting converted, right?

Reply:
Yes @hferentschik i am planning to convert all the existing tests.

Reply:
> This doesn't look like actively maintained because of pending PR and issues. Let make sure we shouldn't hit any blocker during the process when start using it.

I am also concerned that the repo did not see any activity in last few months. Do we have any alternative? Should we reach out to the maintainer?

Reply:
Based on the closed PRs, the last PR merged happened a month ago  https://github.com/stretchr/testify/pulls?q=is%3Apr+is%3Aclosed.

I think this is pretty popular project and it is hard to find any altertive to this handy package.